setup.sh

```diff
@@ -282,17 +282,22 @@ if [ "$BUILD_CONFIG" = "y" ]; then
       }
     }"
 
-    printf ${PMPT} "Does the following configuration appear correct? If yes, configuration will be written to ${CONF_OUTPUT_PATH}: y/n"
-    printf ${GREEN} "${CONFIG_OUTPUT}"
-    read -r GOOD_TO_GO
-    if [ "$GOOD_TO_GO" = "y" ]; then
-      CONFIGURED=1
-      mkdir -p "$OUTPUT_DIR"
-      echo "$CONFIG_OUTPUT" | tee "$CONF_OUTPUT_PATH" >/dev/null
-      chmod 745 "$OUTPUT_DIR"
-      chmod 644 "$CONF_OUTPUT_PATH"
-      printf ${GREEN} "Configuration has been successfully written to ${CONF_OUTPUT_PATH}"
-    fi
+    while [ "$CONFIRMED" != 1 ]; do
```

Comment:
If user enters "n" then it will just keep asking that question without letting them to edit/correct?

Reply:
If they enter 'n' it should go back to the beginning

If they enter anything else than 'y' or 'n' it will ask the question again.

I'm mainly doing this because I have had an issue myself where I accidentally press enter and I have to enter everything over again.

Reply:
I suppose experienced users of device client will just create a config file from a template but for those who are trying it out for the first time, maybe this will help them in case they press enter by accident